cadc-vos-server/src/main/java/org/opencadc/vospace/server/pkg/VospacePackageRunner.java

```diff
@@ -506,9 +557,20 @@ private PackageItem doChildNode(Node child) {
             } catch (Exception e) {
                 log.info(String.format("%s: %s%s", e.getClass().getName(), child.getName(), CONTINUING_PROCESSING));
             }
+            log.debug("doChildNode().end");
             return packageItem;
         }
 
     }
 
+    public class RelativeNode {
+        public ContainerNode node;
+        public String path;
```

Comment:
if this is `parentPath` please make it clear

and if so reverse the order of ctor args: if it is parent, child (to be appended in some logical sense) it should be in that order

Reply:
It's a class for a container node, and the path from the target to the node.
The relative path use is confusing because it's not consistent. It means something different for a container node compared to a data or link node. And as I think about it, I don't think it needs to. It could consistently be the path from the target to the parent node, for all nodes.
See if I can make that change.